elife/php7.sls

```diff
@@ -91,7 +91,7 @@ install-composer:
         - require:
             - cmd: php
             - environ: composer-home
-            - environ: composer-auth
+            - composer-auth
```

Comment:
For reference https://docs.saltstack.com/en/latest/ref/states/requisites.html#require look for `2016.3`